Linphone/App.xaml.cs

```diff
@@ -249,12 +278,13 @@ void OnNavigationFailed(object sender, NavigationFailedEventArgs e)
         /// <param name="e">Details about the suspend request.</param>
         private void OnSuspending(object sender, SuspendingEventArgs e)
         {
-            var deferral = e.SuspendingOperation.GetDeferral();

```

Comment:
Why does not anything here?

Reply:
As i told you in our recent call, it raises on suspension situations (e.g minimizing the app, .... ).
Linphone developers were to force the phone to unregister itself on each suspension events.
We removed this disturbing behavior right here.  
I can completely remove this handler as you may want. 
I kept it for future possible needs.

Reply:
So please remove comment code.